brainfuck/executor.go

```diff
@@ -0,0 +1,54 @@
+package brainfuck
+
+func Interpret(program string) {
+	exec := compile(program)
+	exec.Execute()
+}
+
+func compile(input string) *program {
+	p := newParser()
+	result := newProgram(p.parse(input), newMemory())
```

Comment:
why result? pls name it properly to understand what it includes

Reply:
Fixed